bintray.json

```diff
@@ -2,14 +2,14 @@
     "package": {
         "name": "openmrs-owa-cohortbuilder",
         "repo": "owa",
-        "subject": "openmrs"
+        "subject": "enkya"
```

Comment:
What does enkya mean?

Reply:
Sorry, I had not rebased. That was the initial organisation I was I was pushing to.
Have fixed that